endpoint.bal

```diff
@@ -0,0 +1,536 @@
+            return err;
+        } else {
+            resourcePath = resourcePath + QUESTION_MARK + urlParams;
+            var httpResponse = check self.twitterClient->get(resourcePath, requestHeaders);
+            var response = check handleResponse(httpResponse);
+            Status[] statuses = [];
+            if (response is json) {
+                return convertToStatuses(<json[]> response);
+            }
+            else {
+                return setResponseError(response);
+            }
+        }
+    }
+
+    # Get a user's last ten tweets.
+    # 
+    # + sinceId - Minimum tweet Id 
+    # + maxId - Maximum tweet Id 
+    # + trimUser - User object including only numerical ID. Omit parameter to receive the complete user object  
+    # + excludeReplies - Include retweets 
+    # + includeEntities - Include entities nodes 
+    # + return - If success, returns Status object array, else returns error
+    @display {label: "Get last ten tweets"} 
+    remote function getLast10Tweets(@display {label: "Minimum Tweet Id"} int? sinceId = (), 
+                                    @display {label: "MaximumTweet Id"} int? maxId = (), 
+                                    @display {label: "Trim user or not"} boolean? trimUser = (), 
+                                    @display {label: "Exclude replies or not"} boolean? excludeReplies = (), 
+                                    @display {label: "Include entities or not"} boolean? includeEntities = ()) 
+                                    returns @tainted @display {label: "Array of Status"} Status[]|error {
+        http:Request request = new;
+        string resourcePath = USER_TIMELINE_ENDPOINT ;
+        int count = 10;
+        string urlParams = COUNT + count.toString();
+        string oauthString = urlParams + AMBERSAND + getOAuthParameters(self.clientId, self.accessToken);
+
+        var requestHeaders = createRequestHeaderMap(request, GET, resourcePath, self.clientId, self.clientSecret,
+            self.accessToken, self.accessTokenSecret, oauthString);
+        if (requestHeaders is error) {
+            error err = error(TWITTER_ERROR,
+                              message = "Error occurred while encoding");
+            return err;
+        } else {
+            resourcePath = resourcePath + QUESTION_MARK + urlParams;
+            var httpResponse = check self.twitterClient->get(resourcePath, requestHeaders);
+            var response = check handleResponse(httpResponse);
+            Status[] statuses = [];
+            if (response is json) {
+                return convertToStatuses(<json[]> response);
+            }
+            else {
+                return setResponseError(response);
+            }
+        }
+    }
+}
+
+# Twitter configuration
+#
+# + clientId - Api Key
+# + clientSecret - Api Secret
+# + accessToken - Access token
+# + accessTokenSecret - Access token secret
+# + clientConfig - Client configuration  
+public type TwitterConfiguration record {
```

Comment:
I think we do not need a doc comment like this for records. However, we need display annotations for each field, and display annotation for the record itself.

Reply:
Removed unnecessary doc comments